### PR TITLE
Suppress autogenerated schemes that conflict with custom scheme names

### PR DESCRIPTION
**Pull request by cgrindel**
Related to #573.

### CODE REVIEW

**Review by brentleyjones — APPROVED** [review]
Can you rename back the generator custom scheme so the fixtures show this off?

File: tools/generator/xcodeproj_targets.bzl [hunk 1]
```diff
@@ -8,7 +8,7 @@ GENERATOR_SCHEME_AUTOGENERATION_MODE = "all"
 def get_xcode_schemes():
     return [
         xcode_schemes.scheme(
-            name = "Generator App",
+            name = "Generator",
```
Comment (brentleyjones) [inline tools/generator/xcodeproj_targets.bzl:11]: I know it doesn't exercise the normalization logic, but I really want this scheme to be named after the product, which is lowercase.

```suggestion
            name = "generator",
```

Comment (cgrindel) [reply tools/generator/xcodeproj_targets.bzl]: Done

**Comment by brentleyjones** [comment]
Can you update the fixtures as well?

**Comment by cgrindel** [comment]
I think that I have the filename case issue resolved. I am seeing it with the original lowercase name.

**Pull request merged**